src/components/CardView.ts

```diff
@@ -0,0 +1,68 @@
+import { Card } from "../deck";
```

Comment:
nice, can we keep either 2 spaces or 4 spaces through each files. I don't mind 2 spaces everywhere

Reply:
Turned them to 4, i will let you decide whether you want it to 4 or 2!